### PR TITLE
[YUNIKORN-126] Add execute permission to executables

### PR DESCRIPTION
**Pull request by kingamarton**

### CODE REVIEW

**Comment by sunilgovind** [comment]
@kingamarton so you changed execution permission for all scripts here?

**Comment by kingamarton** [comment]
> @kingamarton so you changed execution permission for all scripts here?

I set executable permission for  build-docker-images.sh and build-release.py and removed the execute permission from LICENSE and NOTICE files

File: tools/build-release.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def setup_base_dir(release_top_path, base_path):
         org = os.path.join(release_top_path, file)
         dest = os.path.join(base_path, file)
         print("copying files: %s ===> %s" % (org, dest))
-        shutil.copyfile(org, dest)
```
Comment (adamantal) [inline tools/build-release.py:84]: Why did you change that?

**Comment by sunilgovind** [comment]
https://github.com/apache/incubator-yunikorn-release/pull/2#issuecomment-630121812
Thanks @kingamarton 
Apart from the comment from @adamantal , I dnt have any other comments.
+1 barring that point.

File: tools/build-release.py (hunk 1 above)
Comment (kingamarton) [reply tools/build-release.py:84]: copyfile will copy only the contents, but the metadata will be skipped, so the permissions will be skipped as well and set some default permissions.
copy2 will copy the metadata as well. see: https://docs.python.org/2/library/shutil.html

Comment (yangwwei) [reply tools/build-release.py:84]: Thanks for the clarification

Comment (adamantal) [reply tools/build-release.py:84]: Thanks!
